TestCases/compliance-level-3/0060-feel-any-function/0060-feel-any-function-test-01.xml

```diff
@@ -27,7 +27,7 @@
 
     <testCase id="004">
         <description>return null if any item not true and not all false</description>
-        <resultNode name="decision004" type="decision">
+        <resultNode errorResult="true" name="decision004" type="decision">
```

Comment:
Case:
https://github.com/dmn-tck/tck/blob/9f8ba50870259c971b5980d031fe73224635be09/TestCases/compliance-level-3/0060-feel-any-function/0060-feel-any-function.dmn#L30

DMN Spec:

> list of Boolean items or argument list of one or more Boolean items

Concur in the result, but I believe is missing the errorFlag to clarify this is an error case